server/hotreloadmiddleware/middleware.js

```diff
@@ -111,9 +111,12 @@ function createSocketStream(opts) {
 			fn(clients[id]);
 		});
 	}
-	var io = require('socket.io')(opts.server);
+	if (opts.socketServer) {
+		var io = opts.socketServer;
+	} else {
+		var io = require('socket.io')(opts.server);
```

Comment:
Let’s move all of the requires to the top of the script so we can easily see the dependencies.